### DISCUSSION TITLE
Hope to support SAS language

### DISCUSSION THREAD

**Discussion by albert-huan** [massCodeIO/massCode#137] [Ideas]
SAS (full name: statistical analysis system, referred to as SAS) is one of the largest private software companies in the world. It is a statistical analysis software developed by North Carolina State University in 1966.

You can refer to this https://github.com/notepad-plus-plus/userDefinedLanguages/blob/master/UDLs/SAS_withPseudocode_byJeffFoarde.xml

Think you very much.

**Comment by sangyuxiaowu** [comment]
massCode use ace editor. support SAS language. maybe you can go there and get help.  https://github.com/ajaxorg/ace

**Reply by antonreshetov** [reply]
In v3 will be SAS support.

![image](https://user-images.githubusercontent.com/17480987/183031821-40b803dd-478a-4778-877a-9944cc49b367.png)
